cmd/kube-controller-manager/app/core.go

```diff
@@ -456,15 +456,17 @@ func startModifiedNamespaceController(ctx context.Context, controllerContext Con
 
 	discoverResourcesFn := namespaceKubeClient.Discovery().ServerPreferredNamespacedResources
 
+	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "namespace controller"))
```

Comment:
```suggestion
	ctx = klog.NewContext(ctx, klog.LoggerWithName(klog.FromContext(ctx), "namespace-controller"))
```

Reply:
Or better, "namespace" as in https://github.com/kubernetes/kubernetes/blob/ca7804fe1374de9621d6806d81a91d82b8f42009/cmd/kube-controller-manager/app/controllermanager.go#L440

It doesn't matter, though, because I'll move the entire name handling out of the controllers into cmd/kube-controller-manager.